frontend/vue/components/Statevector/StatevectorBinaryEncoding.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <StatevectorBaseWidget
+    :number-of-elements="4"
+    :amplitude-disk-data="amplitudeDiskData"
+  >
+    <template #default>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Convallis vitae elit fermentum sit. Pellentesque congue risus sed condimentum viverra dolor. Sapien nunc vulputate posuere vel.
+    </template>
+    <template #input-elements>
+      <bx-checkbox
+        v-for="(value, index) in inputData"
+        :key="index"
+        class="statevector-binary-encoding__input-data__slider"
+        :checked="value"
+        :label-text="inputLabels[index]"
+        @bx-checkbox-changed="inputChangedHandler($event, index)"
+      />
+    </template>
+  </StatevectorBaseWidget>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue-demi'
+import { Amplitude } from '../AmplitudeDisk/amplitude'
+import { randomValue } from '../../../ts/utilities'
+import StatevectorBaseWidget from './StatevectorBaseWidget.vue'
+import 'carbon-web-components/es/components/checkbox/checkbox.js'
+
+const numberOfElements = 4
+
+export default defineComponent({
+  name: 'StatevectorBinaryEncoding',
+  components: {
+    StatevectorBaseWidget
+  },
+  data () {
+    return {
+      inputData: [] as boolean[],
+      inputLabels: ['00', '01', '10', '11'],
+      amplitudeDiskData: [] as Amplitude[]
+    }
+  },
+  mounted () {
+    this.inputData = Array.from({ length: numberOfElements }, _ => randomValue(1, 0, 1) === 1) as boolean[]
+    this.amplitudeDiskData = this.transformFunction(this.inputData)
+  },
+  methods: {
+    transformFunction (inputValues: boolean[]) : Amplitude[] {
+      /*
```

Comment:
are we saving this code for future use?  or is it removable?

maybe we can leave a note in the comment as well...

Reply:
You are right, we can remove it.